[
    {
        "cve": "CVE-2025-20281",
        "poc_github": true,
        "full_name": "abrewer251/CVE-2025-20281-2-Citrix-ISE-RCE",
        "owner_login": "abrewer251",
        "owner_id": 150287770,
        "owner_html_url": "https://github.com/abrewer251",
        "owner_avatar_url": "https://avatars.githubusercontent.com/u/150287770?v=4",
        "html_url": "https://github.com/abrewer251/CVE-2025-20281-2-Citrix-ISE-RCE",
        "description": "Unauthenticated Python PoC for CVE-2025-20281 RCE against Cisco ISE ERS API",
        "fork": false,
        "created_at": "2025-06-27T21:07:27Z",
        "updated_at": "2025-07-01T14:36:18Z",
        "pushed_at": "2025-06-27T21:08:31Z",
        "stargazers_count": 12,
        "watchers_count": 12,
        "has_discussions": false,
        "forks_count": 3,
        "allow_forking": true,
        "is_template": false,
        "web_commit_signoff_required": false,
        "topics": "[]",
        "visibility": "public",
        "forks": 3,
        "watchers": 12,
        "score": 0,
        "subscribers_count": 0
    }
]